rskj-core/src/test/java/co/rsk/peg/P2shErpFederationTest.java

```diff
@@ -137,29 +160,53 @@ void getPowPegAddress_mainnet() {
         assertEquals(expectedAddress, p2shErpFederation.getAddress());
     }
 
-    private void test_getRedeemScript(BridgeConstants bridgeConstants) {
-        List<BtcECKey> defaultKeys = bridgeConstants.getGenesisFederation().getBtcPublicKeys();
-        List<BtcECKey> emergencyKeys = bridgeConstants.getErpFedPubKeysList();
-        long activationDelay = bridgeConstants.getErpFedActivationDelay();
+    @ParameterizedTest()
+    @MethodSource("spendFromP2shErpFedArgsProvider")
+    void spendFromP2shErpFed(
```

Comment:
Do you think would be worth testing erp keys trying to spend before the CSV value condition is reached?

Reply:
We can not do that in an unit test. It is the bitcoin node that validates that condition